modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -30,6 +30,9 @@ You can also use the Agent-based Installer to generate or accept Zero Touch Prov
 ----
 # install-config.yaml
 apiVersion: v1
+metadata:
```

Comment:
Added based [this](https://redhat-internal.slack.com/archives/C01V1DP387R/p1707932602477269) Slack thread.

Reply:
Add a "kind:" parameter? [Example](https://docs.openshift.com/container-platform/4.14/installing/installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.html#:~:text=Example%20mce_namespace.yaml-,apiVersion%3A%20v1,-kind%3A%20Namespace)

Reply:
It is optional and is not used internally in the code. So, no need to add it.